src/plugins/visualizations/public/expressions/visualization_renderer.tsx

```diff
@@ -33,6 +33,7 @@ export const visualization = () => ({
     const visType = config.visType || visConfig.type;
 
     const vis = new ExprVis({
+      title: config.title,
```

Comment:
Seems to be unnecessary line

Reply:
It may be unnecessary for this case, but `ExprVis` has a `title` prop already configured, the `title` position was not in the right location so I fixed while there.

Reply:
You are right, but the `config` doesn't have a title in any kind of visualization.
The config is the result of `fn` of a visualization (in case of table vis that's the result of `table_vis_fn.ts` -> `fn` function).
If you check configs of other visualizations, you'll see that any config doesn't have a title.
So this line do nothing